{"uid":"7bb85890889b3642","name":"test_page_title","fullName":"tests.test_example.ExampleTestCase#test_page_title","historyId":"1f00af4742c2562b8d24e8127e7405ec","time":{"start":1755770771850,"stop":1755770774043,"duration":2193},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//34.245.153.13%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%2034.245.153.13%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","statusTrace":"self = <test_example.ExampleTestCase testMethod=test_page_title>\n\n    def test_page_title(self):\n>       self.home.open(OLLAMA_URL)\n\ntests/test_example.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/home_page.py:21: in open\n    self.driver.get(url)\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:483: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:458: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7efd97d458d0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"Reached error page: about:neterror?e=connection....sys.mjs:156:19\\\\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//34.245.153.13%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%2034.245.153.13%3A3000.\nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nE       UnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\nE       checkReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nE       onNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nE       emit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nE       receiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\n\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":5,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase","time":{"start":1755770768765,"stop":1755770768766,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase::0","time":{"start":1755770776252,"stop":1755770776252,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_example"},{"name":"subSuite","value":"ExampleTestCase"},{"name":"host","value":"pkrvmqc4gcfdwos"},{"name":"thread","value":"2617-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_example"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"ad16f16ec8601352","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.127)\nStacktrace:\n#0 0x55d7746c401a <unknown>\n#1 0x55d774163a70 <unknown>\n#2 0x55d77415acf1 <unknown>\n#3 0x55d77414c099 <unknown>\n#4 0x55d77414dde0 <unknown>\n#5 0x55d77414c43e <unknown>\n#6 0x55d77414bdaa <unknown>\n#7 0x55d77414ba85 <unknown>\n#8 0x55d7741498ee <unknown>\n#9 0x55d77414a0ca <unknown>\n#10 0x55d774166f99 <unknown>\n#11 0x55d774201d65 <unknown>\n#12 0x55d7741db3e2 <unknown>\n#13 0x55d77420114f <unknown>\n#14 0x55d7741db1b3 <unknown>\n#15 0x55d7741a759b <unknown>\n#16 0x55d7741a8971 <unknown>\n#17 0x55d7746891eb <unknown>\n#18 0x55d77468cf39 <unknown>\n#19 0x55d7746702c9 <unknown>\n#20 0x55d77468dae8 <unknown>\n#21 0x55d774654baf <unknown>\n#22 0x55d7746b10a8 <unknown>\n#23 0x55d7746b1286 <unknown>\n#24 0x55d7746c2ff6 <unknown>\n#25 0x7f2b1989caa4 <unknown>\n#26 0x7f2b19929c3c <unknown>","time":{"start":1755770771344,"stop":1755770771938,"duration":594}},{"uid":"d341669909e59c65","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//34.245.153.13%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%2034.245.153.13%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1755770770223,"stop":1755770772422,"duration":2199}},{"uid":"b8af07f18ad8b260","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//34.245.153.13%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%2034.245.153.13%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1755770770118,"stop":1755770772835,"duration":2717}},{"uid":"985de5299b8b0486","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.127)\nStacktrace:\n#0 0x560fd937101a <unknown>\n#1 0x560fd8e10a70 <unknown>\n#2 0x560fd8e07cf1 <unknown>\n#3 0x560fd8df9099 <unknown>\n#4 0x560fd8dfade0 <unknown>\n#5 0x560fd8df943e <unknown>\n#6 0x560fd8df8daa <unknown>\n#7 0x560fd8df8a85 <unknown>\n#8 0x560fd8df68ee <unknown>\n#9 0x560fd8df70ca <unknown>\n#10 0x560fd8e13f99 <unknown>\n#11 0x560fd8eaed65 <unknown>\n#12 0x560fd8e883e2 <unknown>\n#13 0x560fd8eae14f <unknown>\n#14 0x560fd8e881b3 <unknown>\n#15 0x560fd8e5459b <unknown>\n#16 0x560fd8e55971 <unknown>\n#17 0x560fd93361eb <unknown>\n#18 0x560fd9339f39 <unknown>\n#19 0x560fd931d2c9 <unknown>\n#20 0x560fd933aae8 <unknown>\n#21 0x560fd9301baf <unknown>\n#22 0x560fd935e0a8 <unknown>\n#23 0x560fd935e286 <unknown>\n#24 0x560fd936fff6 <unknown>\n#25 0x7f3af909caa4 <unknown>\n#26 0x7f3af9129c3c <unknown>","time":{"start":1755770766676,"stop":1755770767295,"duration":619}},{"uid":"b9ddfe5bd7660741","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.127)\nStacktrace:\n#0 0x5651182b301a <unknown>\n#1 0x565117d52a70 <unknown>\n#2 0x565117d49cf1 <unknown>\n#3 0x565117d3b099 <unknown>\n#4 0x565117d3cde0 <unknown>\n#5 0x565117d3b43e <unknown>\n#6 0x565117d3adaa <unknown>\n#7 0x565117d3aa85 <unknown>\n#8 0x565117d388ee <unknown>\n#9 0x565117d390ca <unknown>\n#10 0x565117d55f99 <unknown>\n#11 0x565117df0d65 <unknown>\n#12 0x565117dca3e2 <unknown>\n#13 0x565117df014f <unknown>\n#14 0x565117dca1b3 <unknown>\n#15 0x565117d9659b <unknown>\n#16 0x565117d97971 <unknown>\n#17 0x5651182781eb <unknown>\n#18 0x56511827bf39 <unknown>\n#19 0x56511825f2c9 <unknown>\n#20 0x56511827cae8 <unknown>\n#21 0x565118243baf <unknown>\n#22 0x5651182a00a8 <unknown>\n#23 0x5651182a0286 <unknown>\n#24 0x5651182b1ff6 <unknown>\n#25 0x7fe55369caa4 <unknown>\n#26 0x7fe553729c3c <unknown>","time":{"start":1755770763671,"stop":1755770764414,"duration":743}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"7bb85890889b3642.json","parameterValues":[]}